runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -628,6 +690,15 @@ static void jitHookInitializeSendTarget(J9HookInterface * * hook, UDATA eventNum
             }
 #endif // defined(J9VM_INTERP_AOT_COMPILE_SUPPORT) && defined(J9VM_OPT_SHARED_CLASSES) && (defined(TR_HOST_X86) || defined(TR_HOST_POWER) || defined(TR_HOST_S390) || defined(TR_HOST_ARM) || defined(TR_HOST_ARM64))
          } // if (TR::Options::sharedClassCache())
+
+#if defined(J9VM_OPT_JITSERVER)
```

Comment:
We have to do the same for the case: "// AOT Body not in SCC, so scount was not set". There are paths that set some value to `count` in that case.
Similarly, we have to cover the case: "// ROM class not in shared class cache (no space or ineligible - maybe because of implicit SCC)"